### PR TITLE
Add H&R Block

### PR DESCRIPTION
**Pull request by legoguy1000**
Adding H&R Block

### CODE REVIEW

**Comment by Carlgo11** [comment]
Thanks for the pull request!

I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
screenshot

**Comment by legoguy1000** [comment]
This is the only documentation I could find which doesn't really help,  https://www.hrblock.com/tax-center/support/online/technical-issues-online/hr-block-security-code/, I've uploaded screenshots below.

image
image
image

File: _data/finance.yml [hunk 1]
```diff
@@ -145,6 +145,14 @@ websites:
       - sms
     doc: https://support.gusto.com/1066223171
 
+  - name: H&R Block
+    url: https://www.hrblock.com/
+    tfa:
+      - sms
```
Comment (Carlgo11) [inline _data/finance.yml]:
```suggestion
```

File: _data/finance.yml [hunk 2]
```diff
@@ -145,6 +145,14 @@ websites:
...
+      - email
+      - software
```
Comment (Carlgo11) [inline _data/finance.yml]:
```suggestion
      - totp
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Based on the screenshots you provided I'd say these are the correct 2fa methods to list.

**Comment by legoguy1000** [comment]
I updated the PR with the changes.  I know in the past I had used SMS to get the code but they may have removed that since.

**Review by Carlgo11 — APPROVED** [review]

**Review by Jawshy — APPROVED** [review]

**Pull request merged**